src/azure-cli/azure/cli/command_modules/storage/_params.py

```diff
@@ -819,6 +819,84 @@ def load_arguments(self, _):  # pylint: disable=too-many-locals, too-many-statem
+        c.extra('in_column_separator', arg_group='Input Delimited Text Configuration',
+                arg_type=column_separator)
+        c.extra('in_quote_char', arg_group='Input Delimited Text Configuration',
+                arg_type=quote_char)
+        c.extra('in_record_separator', arg_group='Input Delimited Text Configuration',
+                arg_type=record_separator)
+        c.extra('in_escape_char', arg_group='Input Delimited Text Configuration',
+                arg_type=escape_char)
+        c.extra('in_has_header', arg_group='Input Delimited Text Configuration',
+                arg_type=has_header)
+        c.extra('out_line_separator',
+                arg_group='Output Json Text Configuration',
+                arg_type=line_separator)
+        c.extra('out_column_separator', arg_group='Output Delimited Text Configuration',
+                arg_type=column_separator)
+        c.extra('out_quote_char', arg_group='Output Delimited Text Configuration',
+                arg_type=quote_char)
+        c.extra('out_record_separator', arg_group='Output Delimited Text Configuration',
+                arg_type=record_separator)
+        c.extra('out_escape_char', arg_group='Output Delimited Text Configuration',
+                arg_type=escape_char)
+        c.extra('out_has_header', arg_group='Output Delimited Text Configuration',
+                arg_type=has_header)
+        c.extra('result_file', help='Specify the file path to save result.')
+        c.ignore('input_config')
+        c.ignore('output_config')
+
+    with self.argument_context('storage blob config-query-format') as c:
+        c.argument('type', arg_type=get_enum_type(['csv', 'json']), help='The serialization format.')
+        c.argument('line_separator', arg_group='Delimited JSON Configuration',
+                   help="The string used to separate records. The default value is '\\n'.")
+        c.argument('column_separator', arg_group='Delimited CSV Configuration',
+                   help="The string used to separate columns. Default to ','.")
+        c.argument('quote_char', arg_group='Delimited CSV Configuration',
+                   help="The string used to quote a specific field. Default to '\"'.")
+        c.argument('record_separator', arg_group='Delimited CSV Configuration',
+                   help="The string used to separate records. Default to '\\n'")
+        c.argument('escape_char', arg_group='Delimited CSV Configuration',
+                   help='The string used as an escape character. Default to empty.')
+        c.argument('has_header', arg_group='Delimited CSV Configuration', arg_type=get_three_state_flag(),
+                   help='Whether the blob data includes headers in the first line. The default value is False, '
+                   'meaning that the data will be returned inclusive of the first line. If set to True, the data '
+                   'will be returned exclusive of the first line.')
```

Comment:
> line_separator = CLIArgumentType(help="The string used to separate records.", default='\n')
column_separator = CLIArgumentType(help="The string used to separate columns.", default=',')
quote_char = CLIArgumentType(help="The string used to quote a specific field.", default='"')
record_separator = CLIArgumentType(help="The string used to separate records.", default='\n')
escape_char = CLIArgumentType(help="The string used as an escape character. Default to empty.", default="")

Can these CLIArgumentTypes be reused here?

Reply:
removed this part